install.sh

```diff
@@ -101,13 +101,13 @@ adjust_bin() {
     # We could add --set-rpath too, but then debugedit (called by rpmbuild) barfs
     # on the result. So use LD_LIBRARY_PATH in the thunk, below.
     patchelf \
-	--set-interpreter "$prefix/libreloc/ld.so" \
+	--set-interpreter "$root/$prefix/libreloc/ld.so" \
 	"$root/$prefix/libexec/$bin"
```

Comment:
It's more consistent but wrong. This won't work at all.

Reply:
Why wouldn't it work?  Isn't `prefix` supposed to be a sub-directory under `root`?

Reply:
Installation has a special mindset, related to chroot. $root is where the installed filesystem is located, and the expectation is that files from $root will be copied to / (or alternatively, we will chroot to $root). This is common in rpm/deb packaging.

The idea is that during packaging, we copy the interpreter to $root/opt/scylladb/libexec, but real installation (when rpm -i runs) copies the file to /opt/scylladb/libexec, so anything that embeds a full path needs to strip $root away from it. As a side effect you can't run until this copy is performed.

If you want a local installtion, use $root=/.